pkg/cdi/spec-dirs.go

```diff
@@ -39,7 +40,7 @@ var (
 	// other Cache instances.
 	DefaultSpecDirs = []string{DefaultStaticDir, DefaultDynamicDir}
 	// ErrStopScan can be returned from a ScanSpecFunc to stop the scan.
-	ErrStopScan = errors.New("stop Spec scan")
+	ErrStopScan = fmt.Errorf("stop Spec scan")
```

Comment:
Can you explain why do we need this?

Reply:
We don't strictly need it per se. `github.com/pkg/errors` should be deprecated AFAICT. `fmt.Errorf("... %w", err)` can be used to replace `errors.Wrap()` and the stock `errors.Is()` can be used instead of the `pkg/errors` (plus anything else that shows up for error handling in go2).

With that rationale, I did not want to add more usage of `github.com/pkg/errors`. So instead I removed it completely from the file I was touching for this PR.

Reply:
Sorry, for the confusion. I was trying to say that we can use [New from the errors module](https://pkg.go.dev/errors#New), so we don't need this change.

Reply:
Ah, sorry, I forgot about/overlooked that. Much better as we don't really do string formatting here. Updated.